src/Items/ItemShears.h

```diff
@@ -68,7 +63,7 @@ class cItemShearsHandler :
 
 	virtual short GetDurabilityLossByAction(eDurabilityLostAction a_Action) override
 	{
-		return 0;
+		return 1;
```

Comment:
Are shears used by each and every action in vanilla? I thought there were exceptions.